src/ModalDialog.js

```diff
@@ -36,7 +42,8 @@ class ModalDialog extends React.Component {
     const dialogClasses = {
       ...getClassSet(bsProps),
       [bsClassName]: false,
-      [prefix(bsProps, 'dialog')]: true
+      [prefix(bsProps, 'dialog')]: true,
+      [prefix(bsProps, 'dialog-centered')]: centered
```

Comment:
lets switch to use `createBootstrapComponent` in all the Modal components instead of the bootstrapUtils, you can check out Button.js for an example on how to do it 👍

Reply:
To start with, I have updated the ModalFooter with createBootstrapComponent. Could you check if that's the recommended way?

Reply:
yup looks great